plugins/pencilblue/controllers/admin/content/navigation/nav_item_form.js

```diff
@@ -63,7 +63,7 @@ module.exports = function(pb) {
             self.setPageName(self.navItem[pb.DAO.getIdField()] ? self.navItem.name : self.ls.g('generic.NEW_NAV_ITEM'));
             self.ts.registerLocal('angular_objects', new pb.TemplateValue(angularObjects, false));
             self.ts.registerLocal('content_type', '{{section.type}}');
-            self.ts.registerLocal('selection_id_field', 'item');
+            self.ts.registerLocal('selection_id_field', 'headline');
```

Comment:
I don't remember why i decided using selection_id_field for filter(now i think angular object will be enough), but this field not using in code. [Why it exists](https://github.com/pencilblue/pencilblue/blob/master/plugins/pencilblue/templates/admin/elements/content_search.html#L2)?